.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: ruby
+rvm:
+  - 2.2
+  - jruby
```

Comment:
These are the versions of Ruby that travis will run our tests against. Is only helpful when publishing a gem.

Can remove `jruby` and change 2.2 to whatever the latest ruby is, Something like `2.4.1`